addons/web/static/src/js/services/data_manager.js

```diff
@@ -104,23 +105,23 @@ return core.Class.extend({
                 // no one should edit it
                 utils.deepFreeze(result.fields);
 
+                // Insert filters, if any, into the filters cache
+                if (options.load_filters) {
```

Comment:
Why not use getFilters defined above?

Reply:
`options.load_filters` is used in the RPC to load the filters along with the view and put them in the filters cache. It is set to false if the cache is already populated even though we still might want to retrieve them.
`getFilters` determines whether the filters should be in the final `fields_view_get` object

Reply:
got a bit confused by this as well

Reply:
maybe check `result.filters` instead